content/docs/start/metrics-parameters-plots.md

```diff
@@ -215,24 +218,24 @@ We are definitely not happy with the AUC value we got so far! Let's edit the
 +  ngrams: 2
 ```
 
-And the beauty of `dvc.yaml` is that all you need to do now is to run:
+The beauty of `dvc.yaml` is that all you need to do now is run:
 
 ```dvc
 $ dvc repro
 ```
 
-It'll analyze the changes, use existing cache of previous runs, and execute only
-the commands that are needed to get the new results (model, metrics, plots).
+It'll analyze the changes, use existing <abbr>run-caches</abbr>, and execute
```

Comment:
There's only one run-cache. Can we rephrase a little? But it's probably not bad to insert the tooltip indeed.